docs/entanglement_forging/explanation/index.rst

```diff
@@ -177,6 +177,44 @@ that do not participate in electronic excitations (i.e. core orbitals or
 those that lie out of symmetry) by removing the bits that correspond to
 them.
 
+.. _Fixing the Hartree-Fock bitstring:
+
+Fixing the Hartree-Fock bitstring
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In some cases, it is possible to increase the accuracy of simulations and speed up
+the execution by bypassing the experiments associated with the first bitstring and
+replacing those results with the Hartree-Fock energy value.
+
+.. code-block:: python
+   :caption: Fixing the HF energy at each iteration
+       from qiskit_nature.second_q.problems import ElectronicStructureProblem
+       from circuit_knitting.forging import EntanglementForgingGroundStateSolver
+
+       problem = ElectronicStructureProblem(...)
+       hf_energy = ...
+
+       solver = EntanglementForgingGroundStateSolver(
+           ansatz=ansatz,
+           hf_energy=hf_energy
+       )
+
+       result = solver.solve(problem)
+
+This setting requires an ansatz that leaves the Hartree-Fock (HF) state
+unchanged with respect to the optimization parameters. As a rule of thumb,
+this can be achieved by restricting entanglement between the qubits representing
+occupied orbitals (bits = 1) in the HF state and the qubits representing
+unoccupied orbitals (bits = 0) in the HF state.
```

Comment:
I don't understand what "bits = 1" and "bits = 0" mean.  Actually, once I typed that, it clicked for me.  maybe "(i.e., bits set to 1)"

Reply:
Yes, bits set to 1 represent an occupied orbital